src/main/resources/templates/fragments.html

```diff
@@ -38,6 +45,12 @@
     </div>
 </div>
 
+<div th:fragment="back-button" class="header-left col-2">
+    <button class="back-button" onclick="history.back()">
+        <img th:src="@{/icons/back-button.svg}" alt="Go Back">

```

Comment:
Please test these two things:

What happens if you refresh the page and then click this button? Does it go back to the previous page or does it stay on the same one?

What happens after you rate a place and then click the button?

If it works as expected then no problem, but I think both will stay on the same page.

Reply:
Works fine after refreshing the page.

Once the site is rated it doesn't work properly, but the next planned step is to redesign the rating function so that it uses javascript to call rest endpoint and doesn't redirect to the same page.